src/app/testing/gene-mocks.ts

```diff
@@ -3278,7 +3280,8 @@ export const geneMock3: Gene = {
   ],
   nominations: null,
   is_adi: false,
-  is_tep: true
+  is_tep: true,
+  resource_url: ''
```

Comment:
Is an empty string an acceptable value for `resource_url` when `is_tep` is true? Just curious based on this statement in the ticket: "When the above condition is satisfied, gene_info.resource_url should always be populated." where the condition is that either `is_tep` or `is_adi` is true.

Reply:
You're right.  I made a change to the original interface since it could be null.